### ISSUE TITLE
Strategy Contracts not appearing

### ISSUE THREAD

**Issue by ArkadyDR** [jrossignol/Strategia#61]
KSP 1.2.2, Strategia v1.5.0, GPP 1.1.0

When activating the planet strategies (eg Iota Probes, Ceti Probes, The Iota Program), the corresponding contract isn't appearing in the contracts list, and so I can't complete the strategy.

I've tried cancelling all other contracts with no luck, so it isn't a Max Contracts issue.

Is there some trick to getting Strategia to work with GPP? From reading the forums it seemed like it was supported and should just work.

Any suggestions on how to debug the install to figure out what has gone wrong?

Edit: I've tried cancelling and re-taking the strategies and the contract still doesn't appear. Should I be able to see it in the mission control building?

**Comment by ArkadyDR** [comment]
Note I'm also using Unmanned Before Manned. Started a new career and put a craft into Gael orbit, then took on the Iota Probes strategy, no contract visible in the mission control building.

Must be an installation issue or a conflicting mod?

**Comment by ArkadyDR** [comment]
OK, so the "Strategia" group of contracts isn't showing up with GPP installed, but it does show up when using the stock planetary system with the UBM mod installed (the only change I made was to install Kopernicus and the GPP mod).

**Comment by ArkadyDR** [comment]
So somehow the GPP version I downloaded is 1.1 but was missing the GPP Strategia optional mod folder, restoring that fixed the issue. Sorry for the goose chase!

Note having a screenshot like this one on the page describing the features would have helped me realise it was missing (I installed Strategia for the first time into GPP and it wasn't clear how it was meant to work).

![kerbal space program 26_03_2017 13_17_49](https://cloud.githubusercontent.com/assets/1075910/24329333/da33ba70-1235-11e7-8692-09a34d1036da.png)
